dist/plugin.json

```diff
@@ -14,18 +14,14 @@
       {"name": "Project site", "url": "https://github.com/foursquare/datasource-plugin-clouderamanager"},
       {"name": "Apache License", "url": "https://github.com/foursquare/datasource-plugin-clouderamanager/blob/master/LICENSE"}
     ],
-    "version": "0.9.2",
-    "updated": "2016-10-21"
+    "screenshots": [],
+    "version": "1.0.0",
+    "updated": "2023-04-19"
   },
-
   "dependencies": {
-    "grafanaVersion": "3.x.x",
+    "grafanaDependency": "^9.3.8",
```

Comment:
Given that you have tried the plugin on Grafana 10, and it works, do you think it would make sense to allow it here as well?